nodejs/Dockerfile

```diff
@@ -9,9 +9,9 @@ ENV NODE_ENV $NODE_ENV
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-COPY package.json /usr/src/app/
+COPY nodejs/package.json /usr/src/app/
 RUN npm install && npm cache clean --force
-COPY server.js /usr/src/app/server.js
+COPY nodejs/server.js /usr/src/app/server.js
```

Comment:
These directory changes are not really needed. We should provide correct context to  docker build directly.

Something like,
```
docker buildx build --platform=$(PLATFORMS) -t $(REPO)/nodejs-env:$(TAG) --push -f nodejs/Dockerfile nodejs/
```